browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -200,6 +200,7 @@ class MarkdownNoteDetail extends React.Component {
             }
             ee.once('list:moved', dispatchHandler)
           })
+          .then(() => ee.emit('list:moved'))
```

Comment:
The issue really is that `dispatchHandler` is never called. So we `emit` the event for the dispatch handler to be run.

I think this is better than just directly calling the dispatch handler (`dispatchHandler()`) because it remains asynch using the event emitter (I believe).

Reply:
I see. I think it is better than calling the dispatch handler.
but, if we delete a note in trash, Boostnote select nothing, like below.
![1 -31-2018 18-18-06](https://user-images.githubusercontent.com/14838850/35614669-374799f8-06b3-11e8-80aa-80edc9ab6e46.gif)
this behavior is not we hope.